.github/workflows/jre8-scan.yml

```diff
@@ -1,11 +1,9 @@
 name: Scan jre/8 Docker image
 
 on:
-  pull_request:
-    branches:
-    - master
+  push:
```

Comment:
Should it be `pull_request`? (wondering if we need it per push)

Reply:
Ah, you're right.
Triggered by the PRs related to this path should be enough.

Reply:
Revised in https://github.com/scalar-labs/docker/pull/4/commits/a76648138f57834ae71353fbf1907a79f496d352